pkg/inspector/image-inspector.go

```diff
@@ -182,6 +179,91 @@ func (i *defaultImageInspector) Inspect() error {
 	return nil
 }
 
+func aggregateBytesAndReport(bytesChan chan int) {
+	var bytesDownloaded int = 0
+	timeOutChannel := time.After(PULL_LOG_INTERVAL_SEC * time.Second)
+	for {
+		select {
+		case bytes, open := <-bytesChan:
+			if !open {
+				return
+			}
+			bytesDownloaded += bytes
+		case <-timeOutChannel:
+			log.Printf("Pulling... %d KB", bytesDownloaded/1024)
+			timeOutChannel = time.After(PULL_LOG_INTERVAL_SEC * time.Second)
```

Comment:
Rather than creating a new channel every 10 seconds it might be more appropriate to use a ticker for this.  https://play.golang.org/p/oLwCQVXcKL